clients/State.ts

```diff
@@ -1,40 +1,21 @@
 import {
   type BaseWallet,
-  getBytes,
   AbiCoder,
   type ParamType,
   keccak256,
+  getBytes,
 } from "ethers";
 
-import {
-  ecsign,
-  toRpcSig,
-  keccak256 as keccak256_buffer,
-} from "ethereumjs-util";
-import { type StateStruct } from "../typechain-types/contracts/Nitro-SCW.sol/NitroSmartContractWallet";
+import { type StateStruct } from "../typechain-types/Nitro-SCW.sol/NitroSmartContractWallet";
```

Comment:
Ugh it's pretty annoying this path keeps jumping back from `"../typechain-types/Nitro-SCW.sol/NitroSmartContractWallet"` to `"../typechain-types/contracts/Nitro-SCW.sol/NitroSmartContractWallet"` :( 

Could you try regenerating your bindings by running ` yarn hardhat clean && yarn hardhat compile`?

Reply:
Why don't we get an error on CI for the "wrong" one?

Reply:
Looks like we need to add that to the github action.

Reply:
That's a good question! Looking at our CI workflow I'm not sure if we actually ever "build" everything and there's no tests that reference the clients. So maybe this code is never built or tested, meaning it doesn't matter if it has a bad import statement.

Reply:
#69

Reply:
[3257618](https://github.com/statechannels/SCBridge-Wallet/pull/57/commits/32576188d1a97f9ed80faaf7e7f2141e68c45f6a)